.changeset/dirty-balloons-prove.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/brevo-admin": minor
+"@comet/brevo-api": minor
+---
+
+Allow manually assigning contacts to a target group
+This is in addition to the existing automatic assignment via filters.
```

Comment:
```suggestion
Allow manually assigning contacts to a target group

This is in addition to the existing automatic assignment via filters.
```

The new line here is necessary to differentiate between title and description

Reply:
[19ddf00](https://github.com/vivid-planet/comet-brevo-module/pull/52/commits/19ddf00d7ed8457a250ae66a85a49849e0de536f)